spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianIframeContentServlet.java

```diff
@@ -0,0 +1,70 @@
+package com.k15t.spark.atlassian;
+
+import com.k15t.spark.base.RequestProperties;
+import com.k15t.spark.base.util.DocumentOutputUtil;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+
+/**
+ * The servlet implementation (or sub-class) to use for SPARK iframe functionality.
+ */
+public abstract class AtlassianIframeContentServlet extends AtlassianAppServlet {
+
+    // AtlassianAppServlet handles some heavy lifting required for living in the plugin servlet environment
+    // eg. finding out the real servlet path and handling caching in that environment
+
+
+    @Override
+    protected String prepareIndexHtml(String indexHtml, RequestProperties props) throws IOException {
+        Document document = Jsoup.parse(indexHtml, props.getUri().toString());
+
+        if (!isDevMode()) {
+            applyCacheKeysToResourceUrls(document, props);
+        }
+
+        prepareIframeContentIndex(document);
+        customizeIframeContentDocument(document);
+
+        document.outputSettings().prettyPrint(false);
+        indexHtml = document.outerHtml();
+        return indexHtml;
+    }
+
+
+    /**
+     * <p>
+     * Serves the index page in iframe content mode
+     * </p>
+     * <p>
+     * The contentWindow part of the iFrameResizer will be added to the document as inline script,
+     * otherwise the document will be left untouched.
+     * </p>
+     *
+     * @param document {@link Document} of the index before processing
+     */
+    private void prepareIframeContentIndex(Document document) throws IOException {
+        // load contentWindow part of the iFrameResizer (inject as inline script), otherwise left the app untouched
```

Comment:
The docs of this method doesn't seem to fit anymore.

Either remove the complete method by moving the three lines into ```prepareIndexHtml()``` or rename the method to something like ```injectIframeResizer()``` etc. and remove the javadocs.